.github/workflows/build.yaml

```diff
@@ -229,12 +229,28 @@ jobs:
               run: |
                   ./scripts/run_in_build_env.sh \
                     "./scripts/build/build_examples.py --no-log-timestamps \
-                       --target linux-x64-all-clusters-ipv6only-clang \
-                       --target linux-x64-chip-tool-ipv6only-clang \
                        --target linux-x64-minmdns-ipv6only-clang \
                        --target linux-x64-rpc-console \
                        build \
                     "
+            - name: Prepare code pregen and ensure compile time pregen not possible
```

Comment:
I don't think "Prepare code pregen" is accurate. Isn't this actually doing the pregeneration?

Additionally you are ensuring that compile time generation is not happening in the next step with the move, right?

Reply:
I will rename. Regarding ensuring compile time generation not happening: I rename codegen.py so that if anything tries to execute it, it will compile fail (and I had that happen during debugging ... so it does work)